left_arm:
  kinematics_solver: trac_ik_kinematics_plugin/TRAC_IKKinematicsPlugin
# kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
# uncomment to use ikfast kinematics plugin
# kinematics_solver: baxter_left_arm_kinematics/IKFastKinematicsPlugin
  solve_type: Distance
  kinematics_solver_attempts: 30
  kinematics_solver_search_resolution: 0.05
  kinematics_solver_timeout: 0.5
right_arm:
  kinematics_solver: trac_ik_kinematics_plugin/TRAC_IKKinematicsPlugin
# kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
# uncomment to use ikfast kinematics plugin
# kinematics_solver: baxter_right_arm_kinematics/IKFastKinematicsPlugin
  solve_type: Distance
  kinematics_solver_attempts: 30
  kinematics_solver_search_resolution: 0.05
  kinematics_solver_timeout: 0.5
